### DISCUSSION TITLE
Please Help with optimizing config for my Xeon Silver 4210

### DISCUSSION THREAD

**Discussion by Chaya89** [swar/Swar-Chia-Plot-Manager#932] [Q&A]
My setup as below;
CUP:  Xeon Silver 4210 
Ram: 32GB
Temp: S70 M.2 PCIe GEN3X4 NVMe 1TB
Final Directory: On NAS

**Comment by davevandyke** [comment]
run 4 x 1tb drives in raid 0 (sudo mdadm -C /dev/md0 /dev/nvme[1-4]n1 -n 4 -l 0 -c 64)
or divide the job below into 4 parts. You might want to increase "stagger_minutes" if it takes more than 15 min to write the file to your NAS.

name: job1
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    temporary_directory: /mnt/ssd
    temporary2_directory:
    destination_directory:
      - /mnt/dst/03
      - /mnt/dst/04
    size: 32
    bitfield: true
    threads: 8
    buckets: 128
    memory_buffer: 4000
    max_concurrent: 9
    max_concurrent_with_start_early: 12
    stagger_minutes: 15
    max_for_phase_1: 4
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false

**Reply by Chaya89** [reply]
Thanks so much for reply bro, here my configuration. I am sure it good or not.

 - name: Chia_Temp
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    **temporary_directory: E:\Temp**
    temporary2_directory:
    **destination_directory: X:\New_Plots**
    size: 32
    bitfield: true
    threads: 8
    buckets: 128
    memory_buffer: 4000
    **max_concurrent: 3**
    **max_concurrent_with_start_early: 7**
    **initial_delay_minutes: 0**
    **stagger_minutes: 0**
    **max_for_phase_1: 1**
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false
 
I only have 1TB PCle to store the temp file. this is create multi plot at the same time or not?

**Reply by chiamillonarie** [reply]
HEY! I have 4x1tb NVME SSD INTEL, How can I run them in raid 0 @davevandyke 

> run 4 x 1tb drives in raid 0 (sudo mdadm -C /dev/md0 /dev/nvme[1-4]n1 -n 4 -l 0 -c 64)
> or divide the job below into 4 parts. You might want to increase "stagger_minutes" if it takes more than 15 min to write the file to your NAS.
> 
> name: job1
> max_plots: 999
> farmer_public_key:
> pool_public_key:
> temporary_directory: /mnt/ssd
> temporary2_directory:
> destination_directory:
> - /mnt/dst/03
> - /mnt/dst/04
> size: 32
> bitfield: true
> threads: 8
> buckets: 128
> memory_buffer: 4000
> max_concurrent: 9
> max_concurrent_with_start_early: 12
> stagger_minutes: 15
> max_for_phase_1: 4
> concurrency_start_early_phase: 4
> concurrency_start_early_phase_delay: 0
> temporary2_destination_sync: false

**Reply by chiamillonarie** [reply]
Nvm just found this https://youtu.be/3-655bm9N0s?t=353
Do u think is better having them all 4 in 1? or 2 and 2 ?

**Comment by davevandyke** [comment]
Oh. I thought you had 4 drives of 1 TB each. More SSD storage will greatly
improve your plotting ability. I would aim for at least 2 TB (preferably 3)
for a 10 core processor. With only 1 TB you will be limited to 3 - 4 plots
at a time.

**Reply by Chaya89** [reply]
thanks bro 3 -4 plots is fine with me. but the problem i not sure with configuration. can you help?

**Comment by davevandyke** [comment]
TL;DR
I would test with using them in a raid-0 and separately. Some people have
reported that non-raided is faster, but I like combining the space to allow
for more plots.


A 1tb drive is ~931GB formatted and a Chia k=32 temp file is 239 GB meaning
that you can conservatively  do 3 plots per drive x 4 drives = 12 plots. A
4 drive raid-0 gives you 3,724 GB and allows for 15 simultaneous plots.
Because a temp file is often less than the full 239 GB you can tune the
values to get more than 15 plots at a time.

**Reply by chiamillonarie** [reply]
OKay! Exactly the math I did I think it compensates the fact that are 4 in 1, even if is slightly slower the plotting side 3 more plots its worth it lets see how long it takes me to plot them, thank you!
